.github/CODEOWNERS

```diff
@@ -12,3 +12,14 @@
 * @signalfx/gdi-specification-approvers @signalfx/gdi-specification-maintainers
 
 .github/CODEOWNERS @signalfx/gdi-specification-maintainers
+
+#####################################################
+#
+# Docs reviewers
+#
+#####################################################
+
+*.md @signalfx/docs
```

Comment:
This would enable ownership of the main specification content in this repository as it is all written in markdown. I don't think the intent is to have the docs teams owners of this, right?